README.md

```diff
@@ -51,7 +51,7 @@ The SAF CLI is the successor to [Heimdall Tools](https://github.com/mitre/heimda
   * [View](#view) - Identify overall security status and deep-dive to solve specific security defects
   * [Validate](#validate) - Verify pipeline thresholds
   * [Generate](#generate) - Generate InSpec validation code, set pipeline thresholds, and generate options to support other saf commands.
-  * [Supplement](#supplement) - Supplement elements that provide contextual information in an HDF file
+  * [Supplement](#supplement) - Supplement elements that provide contextual information in the Heimdall Data Format results JSON file such as `passthrough` or `target`
```

Comment:
We already define HDF earlier in the document, I don't think we need to say it in full here.  HDF is by default a results json.  Many of the explanations here are shorter than their descriptions used within their actual sections, so I think removing the 'such as' part would help in making the TOC section here not as long.

Reply:
Talked with Eugene - shorten to HDF, keep such as

Reply:
done